Gemfile

```diff
@@ -118,6 +119,12 @@ gem 'dogapi', '~> 1.45'
 # This gem can be removed once we upgrade to Ruby 3.1.
 gem 'net-http'
 
+# Deployment
+gem 'capistrano', '~> 3.19', '>= 3.19.1'
```

Comment:
All capistrano and capistrano-related gems should be in the `development` group.

https://github.com/capistrano/rails/blob/master/README.md#installation